dist/dicom/src/structuredReportToHtml.ts

```diff
@@ -101,10 +105,12 @@ async function structuredReportToHtml(
     args.push('--add-document-type')
   }
   if (options.cssReference) {
-    args.push('--css-reference', '2')
+    args.push('--css-reference', '1')
+    inputs.push({ type: InterfaceTypes.TextStream, data: { data: options.cssReference } })
   }
   if (options.cssFile) {
-    args.push('--css-file', '3')
+    args.push('--css-file', '2')
```

Comment:
If `cssReference` was not passed, this would not be `2` -> let's keep track of the number of PipelineInput's that are used, increment the integer, then convert to string.

Reply:
Since we are using `inputs.push` for every such parameter, how about using `inputs.length` itself as the counter?

Reply:
Yes, good idea :zap: :bulb: